### ISSUE TITLE
setHitTestVisible would cause the titlebar not working

### ISSUE THREAD

**Issue by zxcv1884** [wangwenx190/framelesshelper#183]
When I use setHitTestVisible the titlebar is not working.

Branch: main 648876b
System: Windows 10.0.19045
Qt: 6.4.1 MSCV2019 64bit

https://github.com/wangwenx190/framelesshelper/blob/648876b6b009937f39216a39b10610c50074b2f2/examples/quick/Window.qml#L48-L62

Replace line 54 to:
```
FramelessHelper.setHitTestVisible(titleBar.minimizeButton);
```

Logs:
```
[2022/11/21 16:08:33.016] <WARNING> "Could not convert argument 0 at"
[2022/11/21 16:08:33.016] <WARNING> 	 "expression for onReady@qrc:/Demo/Window.qml:54"
[2022/11/21 16:08:33.017] <WARNING> qrc:/Demo/Window.qml:54: TypeError: Passing incompatible arguments to C++ functions from JavaScript is not allowed.
```

**Comment by wangwenx190** [comment]
Can you check whether this issue is fixed by 3961ecb or not? Thanks!

**Comment by zxcv1884** [comment]
It's works! thank you
